### PR TITLE
Changes virus healing to conversion

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]: #
:cl: nicbn
tweak: Now virus healing converts one damage to another.
/:cl:

Closes #32387

I'm open to better conversion ideas.

Currently:
toxin -> 5 respiration
brute -> 2 toxin
burn -> toxin
brute -> burn

### CODE REVIEW

**Comment by Arianya** [comment]
Could use hunger, so a symptom for brute, burn and toxin each that makes you hungrier but heals the related damage at a slow-but-faster-then-vitamins rate, but you'd have to get rid of Weight Gain/Weight Even

**Comment by PKPenguin321** [comment]
You would not have to get rid of weight gain

File: code/datums/diseases/advance/symptoms/conversion.dm [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+	Little bit hidden.
+	Lowers resistance tremendously.
+	Decreases stage speed tremendously.
+	Decreases transmittablity temrendously.
+	Fatal Level.
+
+Bonus
+	Heals brute damage slowly over time.
+	Deals brute damage slowly over time.
```
Comment (PKPenguin321) [inline code/datums/diseases/advance/symptoms/conversion.dm]: deals burn*

**Comment by PKPenguin321** [comment]
Could use a better OP so it's understandable what it does at a glance

**Comment by Ratvar** [comment]
brute + burn => toxin virus seems like a win if you secure charcoal production

**Comment by ShizCalev** [comment]
How does toxin o2 play with self respiration?

**Comment by nicbn** [comment]
Self-respiration heals 7 oxyloss, toxin to respiration deals 5 x power damage.

File: code/datums/diseases/advance/symptoms/conversion.dm [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	Heals burn damage slowly over time.
+	Intoxicates mob's bloodstream.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/conversion/burn
+
+	name = "Regenerative Compensation"
+	desc = "The virus produces toxins and hormones which estimulates the body to regenerate burnt tissues."
```
Comment (SgtxRootbeer) [inline code/datums/diseases/advance/symptoms/conversion.dm]: estimulates

File: code/datums/diseases/advance/symptoms/conversion.dm [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+	Heals brute damage slowly over time.
+	Intoxicates mob's bloodstream.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/conversion/brute
+
+	name = "Mending Compensation"
+	desc = "The virus heals bruises, secretating large amounts of toxins in the proccess."
```
Comment (SgtxRootbeer) [inline code/datums/diseases/advance/symptoms/conversion.dm]: "secretating"
should be "secreting"

**Comment by fludd12** [comment]
Maybe add stamina drain and lessen the oxyloss from toxin, so the healing is much less effective midcombat, but still valuble for general use?

**Comment by XDTM** [comment]
Brute healing is doing 2x of tox damage, a damage type that has even less available cures? I think nobody would even bother tbh.

I stand for #32432

**Comment by nicbn** [comment]
This is not an alternative to #32432
One PR does not exclude another

**Comment by SyruseEln** [comment]
So if a get a healing virus and take like 40 damage of any kind i suffocate to death? Am i reading this right?

**Comment by nicbn** [comment]
Yes.
If you don't like it don't take the virus.

**Comment by nicbn** [comment]
Yes. This is not supposed to be free healing, this is a tradeoff.
You regenerate a little, but you won't resist lots of damage.

**Comment by AnonymousNow** [comment]
Why has passive healing been nerfed into the ground so hard these past few years?

First a good change in its visibility, then the hivelord core implant being molested, and now it's being attacked on all sides.

Straight tradeoffs for healing viruses make little sense. Closest comparisons in things like cult magic trade more damage of one type for less of another. The idea of using mechanics like oxyloss and hunger makes more sense.

**Comment by Bawhoppen** [comment]
Passive healing removes so many aspects of the game. And if you think it's been nonstop nerfed in the past few years, then just remember how many new powerful healing things have been added.

**Comment by XDTM** [comment]
I can agree with having both conversion and conditional healing, although i think conversion should deal equal or less damage of the new type imo. Currently the brute to tox + flesh eating doubles the damage and makes it even harder to cure

**Comment by nicbn** [comment]
In defense of my PR, I want to say that this is just a feature, this doesn't mean that people are supposed to use it. So, if you think that it's not worth it to use compensation, you can just not use it. Unless the virology makes a virus airborne, then you can just sit and cry.

**Comment by XDTM** [comment]
My point was that it's technically harmful, on the brute one at least. It has its uses, but for antags.

**Comment by nicbn** [comment]
Actually you could just use brute -> burn -> toxin -> 5 respiration.
If you take less than 20 damage you will heal just fine.

**Comment by KorPhaeron** [comment]
XDTMs version got merged, we'll try that for a while and reconsider this if his doesn't work out

**Comment by nicbn** [comment]
>This is not an alternative to #32432
>One PR does not exclude another

**Pull request closed without merging**